homeassistant/components/unifiprotect/strings.json

```diff
@@ -91,6 +91,36 @@
         }
       }
     },
+    "rtsp_disabled_readonly": {
+      "title": "RTSPS is disabled on camera {camera}",
+      "fix_flow": {
+        "step": {
+          "start": {
+            "title": "RTSPS is disabled on camera {camera}",
+            "description": "RTSPS is disabled on the camera {camera}. RTSPS is required to be able to live stream your camera within Home Assistant. If you do not enable RTSPS, it may create an additional load on your UniFi Protect NVR as any live video players will default to rapidly pulling snapshots from the camera.\n\nPlease [enable RTSPS]({learn_more}) on the camera and then comeback and confirm this repair."
+          },
+          "confirm": {
+            "title": "[%key:component::unifiprotect::issues::rtsp_disabled_readonly::fix_flow::step::start::title%]",
+            "description": "Are you sure you want to leave RTSPS disabled for {camera}?"
+          }
+        }
+      }
+    },
+    "rtsp_disabled_writable": {
+      "title": "RTSPS is disabled on camera {camera}",
+      "fix_flow": {
+        "step": {
+          "start": {
+            "title": "[%key:component::unifiprotect::issues::rtsp_disabled_readonly::fix_flow::step::start::title%]",
+            "description": "RTSPS is disabled on the camera {camera}. RTSPS is required to be able to live stream your camera within Home Assistant. If you do not enable RTSPS, it may create an additional load on your UniFi Protect NVR as any live video players will default to rapidly pulling snapshots from the camera.\n\nYou may manually [enable RTSPS]({learn_more}) on your selected camera quality channel or Home Assistant can automatically enable the highest quality channel for you. Confirm this repair once you have enabled the RTSPS channel or if you want Home Assistant to automatically enable the highest quality."
```

Comment:
```suggestion
            "description": "RTSPS is disabled on the camera {camera}. RTSPS is required to live stream your camera within Home Assistant. If you do not enable RTSPS, it may create an additional load on your UniFi Protect NVR as any live video players will default to rapidly pulling snapshots from the camera.\n\nYou may manually [enable RTSPS]({learn_more}) on your selected camera quality channel or Home Assistant can automatically enable the highest quality channel for you. Confirm this repair once you have enabled the RTSPS channel or if you want Home Assistant to enable the highest quality automatically."
```